proposals/open-telemetry/microprofile-opentelemetry-tracing.asciidoc

```diff
@@ -18,17 +18,25 @@ endif::[]
 
 == Introduction
 
-TODO
+In cloud-native technology stacks, distributed and polyglot architectures are the norm. Distributed architectures
+introduce a variety of operational challenges including how to solve availability and performance issues quickly.
+These challenges have led to the rise of observability.
 
-== Rationale
+Telemetry data is needed to power observability products. Traditionally, telemetry data has been provided by either
+open-source projects or commercial vendors. With a lack of standardization, the net result is the lack of data
+portability and the burden on the user to maintain the instrumentation.
 
-TODO
+The https://opentelemetry.io[OpenTelemetry] project solves these problems by providing a single, vendor-agnostic
+solution.
```

Comment:
I suggest we add some rationale on why we need to do MP OpenTelemetry - I think we need to add the trace id propagation

Reply:
The trace id propagation is already included in one of the links to the OTel Spec in the Architecture section.